app/helpers/application_helper/button/smart_state_scan.rb

```diff
@@ -10,6 +8,14 @@ def check_smart_roles
     end
   end
 
+  def calculate_properties
+    super
+    if disabled?
```

Comment:
@PanSpagetka, @romanblanco, @vecerek .. feels like this is something we may need on all the buttons, not just this one, WDYT?

(Currently, `onwhen` wins over `disabled`, and doesn't show the right disabled message.)